flink-jepsen/src/jepsen/flink/mesos.clj

```diff
@@ -24,11 +24,35 @@
             [jepsen.os.debian :as debian]
             [jepsen.flink.zookeeper :refer [zookeeper-uri]]))
 
+;;; runit process supervisor (http://smarden.org/runit/)
+;;;
+;;; We use runit to supervise Mesos processes because Mesos uses a "fail-fast" approach to
+;;; error handling, e.g., the Mesos master will exit when it discovers it has been partitioned away
+;;; from the Zookeeper quorum.
+
+(def runit-version "2.1.2-3")
+
+(defn create-supervised-service!
+  "Registers a service with the process supervisor and starts it."
+  [service-name cmd]
+  (let [service-dir (str "/etc/sv/" service-name)
+        run-script (str service-dir "/run")]
+    (c/su
+      (c/exec :mkdir :-p service-dir)
+      (c/exec :echo (clojure.string/join "\n" ["#!/bin/sh" cmd]) :> run-script)
```

Comment:
It's generally considered best practice for runit units to include an `exec 2>&1` line, and to prefix your command with `exec ...`. So I'd suggest updating this line accordingly; `["#!/bin/sh" "exec 2>&1" (str "exec " cmd)]`

Reply:
I'll fix this.